src/Npgsql/NpgsqlBinaryImporter.cs

```diff
@@ -358,6 +364,19 @@ async Task WriteRow(bool async, params object[] values)
                 await Write(value, async);
         }
 
+        /// <summary>
+        /// Checks if the column to be written belongs to the allowed columns for this importer
+        /// </summary>
+        /// <param name="targetColumnIndex">The target column index to be written. <see cref="_column"/></param>
+        /// <param name="currentNumColumns">The number of columns for this binary importer. <see cref="NumColumns"/></param>
```

Comment:
While documentation is a good thing, there is no need in it for private small methods. Code documents itself in such cases.

Reply:
Sure, I was not sure if I should add it or not.. but I agree. Removed.